MapboxNavigation/FeedbackViewController.swift

```diff
@@ -106,7 +106,7 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
     }
     
     /**
-     Initialize a new FeedbackViewController from an `EventsManager`.
+     Initialize a new FeedbackViewController from an `NavigationEventsManager`.
```

Comment:
Very minor grammar nit: "an" -> "a"

Reply:
Don't forget to fix up [this change](https://github.com/mapbox/mapbox-navigation-ios/pull/1786/files#diff-a25e8b3bfe9ceec368b83a004a6d1708R118) below